x/feemarket/keeper/params.go

```diff
@@ -63,13 +65,19 @@ func (k Keeper) SetParams(ctx sdk.Context, params types.Params) error {
 
 // GetBaseFeeEnabled returns true if base fee is enabled
 func (k Keeper) GetBaseFeeEnabled(ctx sdk.Context) bool {
-	params := k.GetParams(ctx)
+	params, err := k.GetParams(ctx)
+	if err != nil {
+		panic(err)
```

Comment:
This should be error handling, not panic.

Reply:
https://github.com/xpladev/ethermint/pull/14/commits/3914a350d0e6bb6fca5a4faa27dd4efe823c7e2a